components/bl-autocomplete-component/src/text-field/buttons.js

```diff
@@ -0,0 +1,32 @@
+import { ButtonClear } from './button-clear';
+import { ButtonPopup } from './button-popup';
+
+export function Buttons(props) {
```

Comment:
Buttons => ButtonContainer
or
button-container => buttons

I would prefer
`ButtonsContainer` and `buttons-container`